packages/wordpress-kit/src/lib/tailwindcssPlugin/Config.ts

```diff
@@ -55,7 +55,6 @@ export const mergeToConfig: Config = {
 		'.wp-block-video',
 		'.wp-block-spacer',
 		'.wp-block-separator',
-		'.wp-block-columns',
```

Comment:
@mitchellmarkoff Could be missing some context here, but I think this change would essentially be the equivalent of reverting https://github.com/pantheon-systems/decoupled-kit-js/pull/383 - so that would be a regression.

Reply:
Ah my bad I should have seen this slipped in when I rebased with canary. Not intentional, will get it back in now.